scripts/india_census/primary_religion_data/preprocess.py

```diff
@@ -22,10 +22,52 @@
 from os import path
 import urllib.request
 from ..common.utils import title_case
-from ..common.generic_base import CensusGenericDataLoaderBase
 
-
-class CensusPrimaryReligiousDataLoader(CensusGenericDataLoaderBase):
+CENSUS_DATA_COLUMN_START = 7
+
+_RELIGION_DCID_MAPPING = {
+    "Hindu":
+        "dcs:Hinduism",
+    "Muslim":
+        "dcs:Islam",
+    "Christian":
+        "dcs:Christianity",
+    "Sikh":
+        "dcs:Sikhism",
+    "Buddhist":
+        "dcs:Buddhism",
+    "Jain":
+        "dcs:Jainism",
+    "Other religions and persuasions":
+        "dcs:IndiaCensus_OtherReligionAndPersuasions",
+    "Religion not stated":
+        "dcs:ReligionNotStated"
+}
+
+TEMPLATE_STAT_VAR = """Node: dcid:{name}
+name: "{description}"
```

Comment:
Curious why we want to add name and description?

For schema-ful stat-vars (like these) if we set the name property, it will be used on SV display, etc.

Reply:
We can remove it. We have used only  `description: "{description}"` in other census imports. We can do the same here,